### PR TITLE
translated the remaining strings

### PR DESCRIPTION
**Pull request by pandu-12345**

### CODE REVIEW

**Review by mlykotom — APPROVED** [review]
Thank you for the contribution!

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
     <!-- Accessibility -->
     <string name="a11y_plant_item_image">Image de plante</string>
-    <string name="gallery_title">(translate me) Photos by Unsplash</string>
-    <string name="gallery_content_description">(translate me) Navigate to gallery screen</string>
+    <string name="gallery_title">Photos par Unsplash</string>
+    <string name="gallery_content_description">Naviguer vers l'écran de la galerie</string>
```
Comment (mlykotom) [inline app/src/main/res/values-fr/strings.xml:47]:
```suggestion
    <string name="gallery_content_description">Naviguer vers l\'écran de la galerie</string>
```

Comment (mlykotom) [reply app/src/main/res/values-fr/strings.xml:47]: This is causing the build issue. You need to fix it first before applying. Thanks!

**Comment by riggaroo** [comment]
Hi, 
Thanks for your contribution, however Sunflower is going to be archived in favour of maintaining a smaller set of up-to-date set of compose-samples, that can be found here: https://github.com/android/compose-samples, as well as snippets that can be found here: https://github.com/android/snippets.

Please feel free to keep your fork up-to-date for your learning, but this repository will be archived shortly.

**Pull request closed without merging**